tests/sharepoint/resources/list_item2.json

```diff
@@ -32,7 +32,7 @@
   "Weight_x0020_after_x0020_Surgery": null,
   "PedigreeName": "Slc6a3-Cre-650102",
   "Breg2Lamb": null,
-  "HeadpostType": "AI Straight Headbar",
+  "HeadpostType": "CAM-style headframe (0160-100-10 Rev A)",
```

Comment:
shouldn't this be "CAM-style" ? I might not understand what this particular file is - but doesn't the client break this into type and part number?